doc/classes/ScrollContainer.xml

```diff
@@ -52,10 +52,22 @@
 			Deadzone for touch scrolling. Lower deadzone makes the scrolling more sensitive.
 		</member>
 		<member name="scroll_horizontal" type="int" setter="set_h_scroll" getter="get_h_scroll" default="0">
-			The current horizontal scroll value.
+			The current horizontal scroll value. 
+			[b]Note:[/b] If you are setting this value in the [code]_ready[/code] function, it needs to be wrapped with [code]set_deferred[/code] since scroll bar [code]max_value[/code] is not updated yet. 
```

Comment:
```suggestion
			[b]Note:[/b] If you are setting this value in the [method Node._ready] function or earlier, it needs to be wrapped with [method Object.set_deferred], since scroll bar's [member Range.max_value] is not initialized yet. 
```